### PR TITLE
Tweak tooltip spinner

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/v3-project-management/issues/161.

### CODE REVIEW

File: app/javascript/components/student/ExerciseStatusChart.tsx [hunk 1]
```diff
@@ -52,6 +52,18 @@ export const ExerciseStatusChart = ({
 
 const DEFAULT_ERROR = new Error('Unable to load information')
 
+const LoadingComponent = () => {
+  const [isShowing, setIsShowing] = useState(false)
+
+  setTimeout(() => {
+    setIsShowing(true)
+  }, 200)
```
Comment (ErikSchierboom) [inline app/javascript/components/student/ExerciseStatusChart.tsx]: On [this page](https://upmostly.com/tutorials/settimeout-in-react-components-using-hooks), they suggest to use a `useEffect` hook for setting and clearing timeouts.

**Review by ErikSchierboom — APPROVED** [review]

**Comment by iHiD** [comment]
I'll leave this unmerged until you've decided about useEffect, but thanks for getting it done! :)

File: app/javascript/components/student/ExerciseStatusChart.tsx (hunk 1 above)
Comment (kntsoriano) [reply app/javascript/components/student/ExerciseStatusChart.tsx]: Ah yes! Good catch.

**Comment by iHiD** [comment]
Thanks @kntsoriano :)

**Pull request merged**